lib/tmdb_handler.rb

```diff
@@ -152,6 +152,19 @@ def tmdb_handler_actor_credit(credit_id)
     @credit = TVActorCredit.parse_results(@credit_results)
   end
 
+  def tmdb_handler_tv_series_search(query)
+    search_url = "#{BASE_URL}/search/tv?query=#{query}&api_key=#{ENV['tmdb_api_key']}"
```

Comment:
This is the exciting new `/search/tv` endpoint. I learned today that they also have a `/search/multi` endpoint, which is neat, but incorporating it into our current workflows would get messy quickly.